src/liblzma/common/memcmplen.h

```diff
@@ -19,6 +19,9 @@
 #	include <immintrin.h>
 #endif
 
+#if defined(_M_X64) // MSVC or Intel C compiler on Windows
```

Comment:
Its interesting that this was only highlighted when threads are disabled. It must have something to do with the `_WIN32_WINNT 0x0600` in `mythread.h`. Either way, the proper include was missing, so thank you for addressing this.

On Windows, <intrin.h> also includes <immintrin.h>, so including both is redundant. Instead, these can be combined into something like this:

```
#if defined(TUKLIB_FAST_UNALIGNED_ACCESS) \
		&& (defined(__INTEL_COMPILER) && defined(_M_X64)) \
			|| (defined(_MSC_VER) && defined(_M_X64)))
#        define <intrin.h>
#else if HAVE_IMMINTRIN_H
#        define <immintrin.h>
#endif
```

Also, please include this as a separate commit from the CI changes. It was discovered because of the CI changes, but the fix is unrelated.

Reply:
> Its interesting that this was only highlighted when threads are disabled.

This was really interesting. I tried to narrow it down, but didn't have juice to finish this investigation. Will apply recommended changes in a separate commit.